roles/openshift_web_console/tasks/install.yml

```diff
@@ -114,15 +114,24 @@
     src: "{{ mktemp.stdout }}/{{ __console_config_file }}"
   register: updated_console_config
 
-- name: Apply the web console template file
-  shell: >
-    {{ openshift_client_binary }} process -f "{{ mktemp.stdout }}/{{ __console_template_file }}"
-    --param API_SERVER_CONFIG="{{ updated_console_config['content'] | b64decode }}"
-    --param IMAGE="{{ openshift_web_console_image_name }}"
-    --param NODE_SELECTOR={{ openshift_web_console_nodeselector | to_json | quote }}
-    --param REPLICA_COUNT="{{ openshift_web_console_replica_count }}"
-    --config={{ mktemp.stdout }}/admin.kubeconfig
-    | {{ openshift_client_binary }} apply --config={{ mktemp.stdout }}/admin.kubeconfig -f -
+- name: Create the template
+  oc_obj:
+    namespace: openshift-web-console
+    name: openshift-web-console
+    kind: Template
+    files:
+    - "{{ mktemp.stdout }}/{{ __console_template_file }}"
```

Comment:
Just confirming this is not an error if the template already exists

Reply:
Added `force: True` to make sure the template and deployment would be overwritten

Reply:
OK, does `force: True` has the same behavior as `oc apply`?

Reply:
`force` would call `oc replace`

Reply:
It seems it has replaced the params and changed the revisions, didn't remove the deployment when I re-ran `config.yml`